app/models/Conference.php

```diff
@@ -26,7 +26,14 @@ public function author()
 
     public function cfpIsOpen()
```

Comment:
Probably can remove it, I left it because I didn't update the call sites, and also because I didn't want to make decisions for you about what you named these methods and how they were used :)

I think a CFP being "open" is a better model of how we think about them, but for some reason `$conference->cfpIsOpen()` feels like a weird method call, so when I was looking for something small to write an example test for I thought I'd try and come up with another name option.